contracts/modules/IdleLiquidityModule.sol

```diff
@@ -17,6 +17,8 @@ import '../../interfaces/actions/IMint.sol';
 
 ///@title Idle Liquidity Module to manage liquidity for a user position
 contract IdleLiquidityModule is BaseModule {
+    ///@dev delta of the amount because its not precise the returned amount swapped
```

Comment:
If possible, change the name of the variable, because it isn't clear what it does, or explain it better in line20 (example: maximum slippage we accept for the swap)
Also, we should specify in which unit it is (amount of the token, percentage,...)